src/PwdResetEmailSentController.js

```diff
@@ -40,6 +40,17 @@ function (Okta, Enums, FormController, FormType) {
       attributes: { 'data-se': 'pwd-reset-email-sent' },
       formChildren: function () {
         return [
+          FormType.View({
+            View: Okta.View.extend({
+              template: '\
+              <span class="overflow-hidden" style="width:0; height:0; display:block" role="alert">{{alert}}</span>\
```

Comment:
is this going to be the way to solve aria screenreader issues? I see the same span element in the other PR you have for security images, so maybe we should create a view that contains this and add it in all the relevant forms.